src/components/Modal.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+
+import {
+  ModalStyle,
+  CloseButton,
+  DataFlex,
+  MoviePic,
+  ModalH1,
+  ModalH3,
+  ModalH5,
+  ModalSmall,
+} from "../styles/Booking.styled";
+
+import close from "../Asset/Xclose.png";
+export default function Modal({
+  setVisible,
+  location,
+  selectedSeats,
+  setSelectedSeats,
+  setBgClick
+}) {
+  const modalClose = () => {
+    setVisible(false);
+    setSelectedSeats([]);
+    setBgClick(true);
+  };
+
+  return (
+    <ModalStyle>
+      <CloseButton onClick={modalClose} src={close} alt="close" />
+      <ModalH1>Thank you for booking</ModalH1>
+      <ModalH3>Order Summary</ModalH3>
+      <article
+        style={{
+          display: "flex",
+          justifyContent: "space-around",
+          marginTop: "48px",
+        }}
+      >
+        <MoviePic src={location.state.movie} alt="movie" />
+
+        <article>
+          <h3>Seats:</h3>
+          <article style={{ display: "flex",height:"47px",width:"202px",overflow:"hidden",whiteSpace:"nowrap",textOverflow:"ellipsis"}}>
+            {selectedSeats.map((seat, index) => {
```

Comment:
Trailing comma's are present in the UI. Refactore this to use join to reduce trailing commas.